lib/shared/types/src/types/config/models/variable.ts

```diff
@@ -39,7 +39,9 @@ export enum VariableType {
 /**
  * Supported variable values
  */
-export type DVCJSON = { [key: string]: string | boolean | number }
+export type DVCJSON = {
+    [key: string]: string | boolean | number | DVCJSON | unknown[] | null
```

Comment:
What does `unknown[] | null` here functionally do? Does this mean we let folks set null / undefined as default values?